tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{36,37,38}-sphinx{2,3},py36-flake8,docs
+envlist = py{36,37,38,39}-sphinx{2,3,4},py{36,37,38,39}-flake8,docs
```

Comment:
There's no need to have flake8 run 4 times, stick to the one variant:
```
py{36,37,38,39}-sphinx{2,3,4},py36-flake8,docs
```

Reply:
ah, makes sense. Haven't used `tox`before,